Composer/packages/electron-server/src/utility/url.ts

```diff
@@ -3,20 +3,28 @@
 
 import lowerCase from 'lodash/lowerCase';
 
-export const parseDeepLinkUrl = deeplinkUrl => {
-  const convertedUrl = new URL(deeplinkUrl);
+import log from './logger';
+const error = log.extend('error');
```

Comment:
There might be a better name for this. `url` or something.

Reply:
Changed it electron-deeplink-url